opennms-base-assembly/pom.xml

```diff
@@ -93,6 +93,10 @@
           </execution>
         </executions>
       </plugin>
+      <!--
```

Comment:
Yes, they are still compiled to ensure that the reports are compileable. This is a build step only.